RestfulDropApi.cs

```diff
@@ -100,7 +101,7 @@ public RestfulDropApi(string pat)
                 {
                     file.Blob.Url = urlDictionary[file.Blob.Id];
                 }
-                return manifest;
+                return new HashSet<VstsFile>(manifest);
```

Comment:
Implies case-sensitive filenames. Assume you want this, just calling out that bit.